notificationworkerlambda/src/test/scala/com/gu/notifications/worker/HarvesterRequestHandlerSpec.scala

```diff
@@ -113,10 +113,10 @@ class HarvesterRequestHandlerSpec extends Specification with Matchers {
         sqsDeliveries
       }
 
-      override val androidBetaDeliveryService: SqsDeliveryService[IO] = (chunkedTokens: ChunkedTokens) => {
-        firebaseSqsDeliveriesCount.incrementAndGet()
-        firebaseSqsDeliveriesTotal.addAndGet(chunkedTokens.tokens.size)
-        sqsDeliveries
+       override val androidBetaDeliveryService: SqsDeliveryService[IO] = (chunkedTokens: ChunkedTokens) => {
```

Comment:
It seems like this part of the code has been reformatted, but it doesn't seem to align with the code above :/